Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -77,9 +77,23 @@ public void useWifiOnly(bool useWifi)
         /// Returns the MccMnc (Mobile Country Code Mobile Network Code)
         /// </summary>
         /// <returns></returns>
-        public string GetCarrierName()
+        public void GetCarrierName()
```

Comment:
Why void instead of string? We can get rid of the carrierName class variable if you return string instead

Reply:
Or I kind of liked it back in the ConfigureMobiledgeXSettings function. Why the move?

Reply:
@franklin-huang-mobiledgex, I spoke with @lgarner and he mentioned that it would be better if setting MobiledgeXIntegration variables happen independent of Location and CarrierName would be better

Reply:
GetCarrierName ought to still return carrierName, otherwise it's updateCarrierName().

Reply:
NameUpdated